config/routes.rb

```diff
@@ -1,7 +1,10 @@
-Rails.application.routes.draw do
+  Rails.application.routes.draw do
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
   devise_for :users
+  resources :movies do
+    resource :watched_movies, only: [:create, :destroy]
+  end
   root "movies#index"
   resources :movies
```

Comment:
ルーティング自体も制限しておいていただけますか。

```suggestion
  resources :movies, only: [:index]
```